### PR TITLE
Upgraded Ruby to 2.2.2

### PR DESCRIPTION
**Pull request by maclover7**
Upgraded Ruby to 2.2.2 due to a [CVE](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/).

cc @arthurnn @indirect

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: ruby
 
 services: memcached
 
-rvm: 2.2.1
```
Comment (dwradcliffe) [inline .travis.yml]: Good catch!

Comment (maclover7) [inline .travis.yml]: Yep, forgot to upgrade Travis in the original commit, so [this build](https://travis-ci.org/bundler/bundler-api/builds/59629929) is expected to fail (as it did).

**Comment by maclover7** [comment]
Build is [green](https://travis-ci.org/bundler/bundler-api/builds/59630022).

**Comment by indirect** [comment]
Thanks! :+1:

**Pull request merged**